lib/auth/access_test.go

```diff
@@ -77,6 +78,8 @@ func TestUpsertDeleteLockEventsEmitted(t *testing.T) {
 	lock, err := types.NewLock("test-lock", types.LockSpecV2{
 		Target: types.LockTarget{MFADevice: "mfa-device-id"},
 	})
+	futureTime := time.Now().UTC().Add(12 * time.Hour)
+	lock.SetLockExpiry(&futureTime)
```

Comment:
This moved the error check too far away from the error. I would move this after the assertion.

Reply:
@zmb3 please see update